pkg/forwarder/forwarder_health.go

```diff
@@ -131,7 +131,7 @@ func (fh *forwarderHealth) healthCheckLoop() {
 func (fh *forwarderHealth) computeDomainsURL() {
 	for domain, apiKeys := range fh.keysPerDomains {
 		apiDomain := ""
-		re := regexp.MustCompile("datadoghq.[a-z]*")
+		re := regexp.MustCompile(`((us|eu)\d\.)?datadoghq.[a-z]+$`)
```

Comment:
Were we a little lazy with the `[a-z]+` part in the regex? In any case, this is just a question, change here looks good to me.

Reply:
Good question: I think we were, wasn't able to find a good justification for it. I left a comment about it in the unit tests, I think this should be addressed as part of the refactor I mentioned.